DeUrgenta/Components/General/CustomCheckboxView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+struct CustomCheckboxView: View {
+    @Binding var isChecked: Bool
+
+    var body: some View {
+        HStack {
+            VStack {
+                Button(action: {
+                    if isChecked {
+                        isChecked = false
+                    } else {
+                        isChecked = true
+                    }
```

Comment:
Is this equivalent to isChecked = !isChecked ?

Reply:
Yes, it's equivalent.